rubbery-raccoons/wired/wired_app/views/detail.py

```diff
@@ -0,0 +1,32 @@
+from django.contrib import messages
+from django.forms.models import model_to_dict
+from django.shortcuts import get_object_or_404, render
+from django.utils import timezone
+
+from wired_app.models import Article, Comment
+from wired_app.forms import CommentForm
+
+
+def article_detail_view(request, slug):
+    art = get_object_or_404(Article, slug=slug)
+    art_dict = model_to_dict(art)
+    art_dict["author"] = art.author.username
+    comments = Comment.objects.filter(parent_post=art).order_by("created")
+
+    if request.method == "POST":  # a comment was submitted
+        form = CommentForm(request.POST)

```

Comment:
I see why you used this view to handle the comment form. But it's confusing for the reader.

I recommend at least adding a comment somewhere explaining that adding a comment is handled by making `POST` requests to the page detail endpoint.

Also -- it looks like users don't have any way to delete or edit their comments. I kind of like it: you have to *commit* to what you post!